<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="40,40">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QWidget" name="display" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,3,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="manage">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>GroupBox</string>
              </property>
              <widget class="QPushButton" name="startStop">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>60</y>
                 <width>80</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
              <widget class="QComboBox" name="gameList">
               <property name="geometry">
                <rect>
                 <x>170</x>
                 <y>60</y>
                 <width>131</width>
                 <height>22</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="keypad"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0,0,0,0,0">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="P1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P1</string>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P2">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P3">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P4">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P6">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P7">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="P8">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>P8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0">
          <item>
           <widget class="QWidget" name="led1" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led2" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led3" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led4" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led5" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led6" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led7" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="led8" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>lua console</string>
         </attribute>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>104</width>
            <height>64</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
